### PR TITLE
Setting up infrastructure for UTF8StringExperiment in order to have full ci and official build support

### PR DESCRIPTION
**Pull request by joperezr**
This isn't ready for merge, I will be adding several commits in order to identify all infra that will be required in order to get both CI and official build support for this first experiment in runtimelab.

cc: @safern to take a look on yml changes. Worth noting that runtimelab.yml is a new yml which the main pipeline of this repo will be based on. runtimelab.yml will be checked into dotnet/runtime when we feel it is at a ready enough state for experiments on this repo.

FYI: @jkotas

### CODE REVIEW

File: eng/pipelines/runtimelab.yml [hunk 1]
```diff
@@ -51,3 +51,33 @@ jobs:
...
+  parameters:
+    jobTemplate: /eng/pipelines/common/global-build-job.yml
+    buildConfig: release
+    platforms:
+    - Linux_x64
+    - Windows_NT_x64
+    jobParameters:
+      testGroup: innerloop
+      nameSuffix: Runtime_Debug
+      buildArgs: -c debug -runtimeConfiguration release
```
Comment (safern) [inline eng/pipelines/runtimelab.yml:68]: These two will build mono, do we want mono in this experiment? 

The default subsets are: clr+mono+libs+installer: https://github.com/dotnet/runtime/blob/34fd21edebf010cfc9a9d685e4ad22d06f916fc1/eng/Subsets.props#L40

Comment (joperezr) [reply eng/pipelines/runtimelab.yml:68]: oh good catch, missed that. No we want coreclr for now only, I'll fix that.

**Comment by jkotas** [comment]
Nit about the branch name: I am wondering whether we need `Experiment` suffix. This suffix would be on all nearly branches in runtimelab that makes it redundant. Would it be better to go with just  `Utf8String`?

**Comment by joperezr** [comment]
Sure, sounds good to me. I’ll send the new PR tomorrow

**Comment by joperezr** [comment]
closing for target branch rename

**Pull request closed without merging**